### ISSUE TITLE
Bildunterschriften werden beim Export nicht berücksichtigt

### ISSUE THREAD

**Issue by wrcg** [RRZE-Webteam/rrze-xliff#20]
die exportierte xliff-Datei enthält keine Bildunterschriften und können damit auch nicht übersetzt werden

**Comment by xwolfde** [comment]
Ja, ist so. Geht leider nicht und die individuelle Überstetzung der BU in Beiträgen ist nicht sinnvoll anhand einer einzelnen übersetzten Seite/Artikel machbar, da die BU aus den Medien selbst kommt.  Wenn dann müssten die Grafik-Attribute, d.h. die Medienbibliotheken übersetzt werden.
Darüber wurde letztes Jahr im Sommer gesprochen.
https://github.com/RRZE-Webteam/rrze-xliff/issues/6
